framework/tlvf/test/tlvf_test.cpp

```diff
@@ -307,14 +307,22 @@ int test_parser()
     auto tlv2 = msg.addClass<tlvLinkMetricQuery>();
     auto tlv3 = msg.addClass<tlvWscM1>();
     // TODO https://github.com/prplfoundation/prplMesh/issues/480
```

Comment:
This todo is about the `tlv3->add_vendor_ext(tlv3->create_vendor_ext())` part, so it should stay close to it. Or rather, it should move to just in front of `tlv4->add_var1(tlv4->create_var1())` which has a copy of the same comment because it also applies to it.